buildlib/jucx-publish.yml

```diff
@@ -0,0 +1,49 @@
+parameters:
+  target: publish
+  temp_cfg: $(System.DefaultWorkingDirectory)/bindings/java/src/main/native/build-java/tmp-settings.xml
+  gpg_dir: $(System.DefaultWorkingDirectory)/bindings/java/src/main/native/build-java/gpg
+
+steps:
+  - bash: |
+      set -eE
+      {
+        echo -e "<settings><servers><server>"
+        echo -e "<id>ossrh</id><username>\${env.SONATYPE_USERNAME}</username>"
+        echo -e "<password>\${env.SONATYPE_PASSWORD}</password>"
+        echo -e "</server></servers></settings>"
+      } > ${{ parameters.temp_cfg }}
+    displayName: Generate temporary config
+
+  - task: DownloadSecureFile@1
+    displayName: Download Secure file
+    inputs:
+      secureFile: sparkucx-secret.gpg
+    name: privateKey
+
+  - task: DownloadSecureFile@1
+    displayName: Download Secure file
+    inputs:
+      secureFile: sparkucx-public.gpg
+    name: publicKey
+
+  - bash: |
+      set -eE
+      TAG=`git describe --tags`
```

Comment:
is it possible we will find more than one tag ?

Reply:
No, for all tags need to use:
```
peterr@MTL-PETERR ~/ucx
$ git show-ref --tag
0b45e29fd3ad7444f2f06508152569bd79b2abb8 refs/tags/hpcx-v2.1
21e500948a82418b83598b47d4c0e431b79a0ff5 refs/tags/hpcx-v2.2
04d391c926c194210e73804c3907505730cb6d6c refs/tags/hpcx-v2.3
....
 git describe --tags returns latest annotated tag:
```
peterr@MTL-PETERR ~/ucx
$ git describe --tags
v1.7.0-rc2
```

Reply:
ok